__tests__/endpoints.test.js

```diff
@@ -348,4 +348,29 @@ describe('NC news endpoint tests',()=>{
             })
         })
     })
+    describe('GETting users', ()=>{
+        describe('GET an array of all users',() => {
+            test('Respond with 200 and an array of objects with all users', () => {
+                return request(app)
+                .get('/api/users')
+                .expect(200)
+                .then(({body}) => {
+                    expect(body.users.length > 0).toBe(true);
```

Comment:
You could use the jest matcher here `toBeGreaterThan` or alternatively look in the test data and check specifically how many there should be